framework-plugins/lisk-framework-http-api-plugin/swagger.yml

```diff
@@ -19,41 +19,22 @@ info:
     # Welcome!
 
     ## Access restrictions
-    The API endpoints are by default restricted to a whitelist of IPs that can be found under `modules.http_api.access.whiteList` in the config.
-    If it is required for your API to be accessible by the public, this can be achieved by changing `modules.http_api.access.public` to `true`.
-    This will allow anyone to make requests to your Lisk Core node.
-    However some endpoints will stay private, hence this means that only a list of whitelisted IPs can successfully make API calls to that particular endpoint;
-    this includes all forging related API calls.
-    By default, only the nodes' local IP is also included in the whitelist, nevertheless it is possible to change the setting in `config` under `modules.http_api.forging.access.whiteList`.
-    For more details, see the [Lisk Core management docs](https://lisk.io/documentation/lisk-core/3.0.0/management/) or the descriptions at the respective endpoint below.
+    The API endpoints are by default restricted to a whitelist of IPs that can be found under `modules.http_api.whiteList` in the config.
+    If it is required for your API to be accessible by the public, this can be achieved by changing `modules.http_api.whiteList` to `[]`.
+    This will allow anyone to make requests to your application.
 
     ## Requests
     Chained filter parameters are logically connected with `AND`.
     `HTTP` is the supported URL schema by default.
-    To enable `HTTPS`, please adjust the `modules.http_api.ssl` section in the config.
 
     ## Responses
     The general response format is JSON (`application/json`).
     The responses for each API request have a common basic structure as shown below:
     ```javascript
     {
         "data": {}, //Contains the requested data.
-        "meta": {}, //Contains additional metadata, e.g. the values of `limit` and `offset`.
```

Comment:
Please keep `meta` consistently for endpoints returning collections you have to have pagination and that requires pagination information.